analyses/cell-type-wilms-tumor-06/README.md

```diff
@@ -0,0 +1,178 @@
+  |PODXL|ENSG00000128567|non-malignant|podocyte|10.1016/j.stem.2019.06.009|NA|
+  |COL6A3|ENSG00000163359|malignant|mesenchymal|10.2147/OTT.S256654|might_also_be_expressed_in_non_malignant_stroma|
+  |THY1|ENSG00000154096|malignant|mesenchymal|10.1093/hmg/ddq042|might_also_be_expressed_in_non_malignant_stroma|
+
+
+### The table GeneticAlterations_metadata.csv contains the following column and information:
+- alteration contains the number and portion of the affected chromosome
+- gain_loss contains the information regarding the gain or loss of the corresponding genetic alteration
+- cell_class is "malignant"
+- cell_type contains the list of the malignant cell types that are attributed to the marker gene, either blastemal, stromal, epithelial or NA if none of the three histology is more prone to the described genetic alteration
+- DOI contains the list of main publication identifiers supporting the choice of the genetic alteration
+- comment can be empty or contains any additional information
+
+|alteration|gain_loss|cell_class|cell_type|DOI|PMID|comment
+|---|---|---|---|---|---|---|
+|11p13|loss|malignant|NA|10.1242/dev.153163|NA|NA|
+|11p15|loss|malignant|NA|10.1128/mcb.9.4.1799|NA|NA|
+|16q|loss|malignant|NA|NA|1317258|Associated_with_relapse|
+|1p|loss|malignant|NA|NA|8162576|Associated_with_relapse|
+|1q|gain|malignant|NA|10.1016/S0002-9440(10)63982-X|NA|Associated_with_relapse|
+
+
+
+## Software requirements
+
+To perform the analysis, run the RMarkdown script in R (version 4.4.1).
+The main packages used are:
+- Seurat version 5
+- Azimuth version 5
+- inferCNV
+- SCpubr for visualization
+- DT for table visualization
+- DElegate for differential expression analysis
+
+### Docker
+
+To build the Docker image, run the following from this directory:
+
+```shell
+docker buildx build . -t openscpca/cell-type-wilms-tumor-06
+```
+
+The image will also be available from ECR: <https://gallery.ecr.aws/openscpca/cell-type-wilms-tumor-06>
+
+To run the container and develop in RStudio Server, run the following **from the root of the repository**, Replacing `{PASSWORD}`, including the curly braces, with a password of your choosing:
+
+```shell
+docker run \
+  --mount type=bind,target=/home/rstudio/OpenScPCA-analysis,source=$PWD \
+  -e PASSWORD={PASSWORD} \
+  -p 8787:8787 \
+  public.ecr.aws/openscpca/cell-type-wilms-tumor-06:latest
+```
+
+This will pull the latest version of the image from ECR if you do not yet have a copy locally.
+
+Navigate to <http://localhost:8787/> and log in with the username `rstudio` and the password you set.
+
+Within RStudio Server, `OpenScPCA-analysis` will point to your local copy of the repository.
+
+#### A note on Apple Silicon
+
+If you are on a Mac with an M series chip, you will not be able to use RStudio Server if you are using a `linux/amd64` or `linux/x86_84` (like the ones available from ECR).
+You must build an ARM image locally to be able to use RStudio Server within the container.
+
```

Comment:
Can you add an H4 section here on Halbritter lab internal development please?

I'd expect that would include how to run the script (this is taking into account some review feedback):

```
./run-podman-internal.py {PASSWORD}
```

This hopefully helps you with your own development if, for example, you go on vacation for two weeks and come back to this! 😄  But it also helps others understand that this bash script isn't for their use.

Reply:
I'll have to ask for some help in the Halbritter lab, add it on my to do list ;)